shoes-core/lib/shoes/common/translate.rb

```diff
@@ -0,0 +1,19 @@
+class Shoes
+  module Common
+    module Translate
+      def translate_left
+        left, = translate
```

Comment:
All right, found the cop: http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/TrailingUnderscoreVariable and disabled that.